packages/rocketchat-lg-sso/server/sso.js

```diff
@@ -66,8 +66,9 @@ function createOrUpdateUserFromJWT(lgJWT) {
   try {
     setAvatarFromGitHubAvatar(rcUser, lgUser)
   } catch (err) {
-    RavenLogger.log(err)
-    logger.warn('could not set avatar from GitHub avatar', err.stack)
+    // this is not fatal, so we'll just warn and move on so as not to interrupt
+    // the sign-in process
```

Comment:
This comment seems unneccessary. It's not really saying anything the code doesn't make clear. Comments have a way of getting out of date, so I prefer to avoid them unless there's no way the code can speak for itself. In this case `catch { warn }` seems pretty straightforward.